guides/source/active_record_encryption.md

```diff
@@ -497,6 +533,10 @@ The digest algorithm used to derive keys. `OpenSSL::Digest::SHA256` by default.
 Supports decrypting data encrypted non-deterministically with a digest class SHA1. Default is false, which
 means it will only support the digest algorithm configured in `config.active_record.encryption.hash_digest_class`.
 
+#### `config.active_record.encryption.compressor`
+
+The compressor used to compress encrypted payloads. It should respond to `compress` and `uncompress`. Default is `Zlib`. You can find more information about compressors in the [Compression](#compression) section.
```

Comment:
> So I hate to ask, especially since I asked you to change it in the first place, but can you update this PR to use that naming convention? I sincerely apologize for not realizing this sooner.
@flavorjones Don't mind. It's not a big deal.

By using `inflate/deflate`, we don't need wrapper class for `zlib`, `ActiveRecord::Encryption::Compressor::Zlib`. Because `Zlib` can respond to `inflate/deflate`. So I think use `Zlib` in document is right. What do you think? 

ref: `ActiveSupport::Cache` https://github.com/rails/rails/blob/b09ae673ab808d13fb104e4f865970dc0fba3761/activesupport/lib/active_support/cache.rb#L305